qemu/tests/cfg/virtio_chardev_trace.cfg

```diff
@@ -0,0 +1,11 @@
+- virtio_chardev_trace:
+    only Linux
+    type = virtio_chardev_trace
+    start_vm = no
+    smp = 8
```

Comment:
`smp` shouldn't be a default value for that some machines maybe have no enough processes. You can get the value of `smp` from dictionary `params`.